src/main/java/ticketservice/jdbc/CustomDataSource.java

```diff
@@ -23,7 +23,7 @@ public static CustomDataSource getInstance() {
             Properties properties = new Properties();
```

Comment:
look like this class already is redundant, suggest you can remove it

Reply:
Fixed in https://github.com/kolosokvit/ticket-service/pull/7/commits/cbe05276c193c9f2fcac08062ddc5474b83378a9